apps/ledger-live-desktop/package.json

```diff
@@ -233,4 +233,4 @@
     "webpackbar": "^5.0.2",
     "yargs": "^15.3.1"
   }
```

Comment:
Hey @pawelnguyen - after [adding the missing dependencies](https://github.com/pawelnguyen/ledger-live/pull/1/files#r888477269) to the LLC package.json (and getting `pnpm build` to succeed in LLC), I got an error running `pnpm build` in LLD related to a missing dependency - one called `fp-ts`.  I was able to resolve that error by adding that dependency here via `pnpm add fp-ts`.  It's strange though - that dependency is only used by @celo/utils, which isn't used directly in LLD.  

I'm not sure what the right solution for this is.  🤔

Reply:
@andyhass so after adding `@celo/connect` and `@celo/utils` I'm not getting any errors when running `pnpm build`. Maybe you could fetch latest changes from this branch and try again?